WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -136,6 +135,8 @@ - (void)viewWillAppear:(BOOL)animated
     self.resultsController.delegate = self;
     [self.resultsController performFetch:nil];
     [self.tableView reloadData];
+    [self updateEditButton];
+    [self maybeShowNUX];
```

Comment:
Quick clarification, that was just me spit-balling, not necessarily asking that the change be made here.  Up to you sir!